duckduckgo/iOS
# setting to prevent links opening associated apps

Task/Issue URL: https://app.asana.com/0/inbox/392891325557408/1145691603022750/1146067154966667
Tech Design URL:
CC:

**Description**:

Added setting to allow user to disable opening associated apps.

**Steps to test this PR**:
1. Check associated apps open by default (e.g. Twitter, Amazon, Facebook)
1. Change setting and check associated apps not open by default
1. Change setting back and check they open again.
1. Ensure other settings still work as expected.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
